test/tests/functional/pbs_mom_dynamic_resource.py

```diff
@@ -162,7 +164,7 @@ def test_res_string_perticular_value(self):
                            id=jid, attrop=PTL_AND)
 
         # Submit a job that requests mom dynamic resource
-        attr = {"Resource_List." + resc_name[0]: '"This is a test"'}
+        attr = {"Resource_List." + resc_name[0]: '\'\"This is a test\"\''}
```

Comment:
why do we need to add single quotes in the string?

Reply:
This is generic change. If we have to pass a string with spaces in custom resources, format should be "qsub -l foo='\"This is a test\"' -- /bin/sleep 100" according to documentation.

Reply:
oh ok, then I think we should fix PTL lib to add quotes instead user do it in tests.

Reply:
@hirenvadalia I have made changes in PTL library to handle strings having spaces.